src/common/select/Select.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import classNames from 'classnames';
+import ReactSelect, { ValueType } from 'react-select';
+
+import './Select.css';
+
+type Option = {
+  value: string;
+  label: string;
+};
+
+type Props = {
+  id?: string;
+  name: string;
+  labelText?: string;
+  value: string | undefined;
+  className?: string;
+  options: Option[];
+  setFieldValue: (name: string, value: string) => void;
+};
+
+const Select = (props: Props) => {
+  // Without this defaultValue is not set / language wont get updated
+  const value = props.options.find(option => option.value === props.value);
+
+  return (
+    <div className={classNames('wrapper', props.className)}>
+      <label htmlFor={props.name} className="select-label">
+        {props.labelText}
+      </label>
+
+      <ReactSelect
```

Comment:
I am not knowledgeable of the design in more depth, but it's currently a filter box while it only has three options. Setting it up in this way makes it more vulnerable for accessibility quirks because the component has to convey a lot more information to screen reader users: current search, search options list, currently selected option etc. With Varaamo, the combination of the extra information broke this component for some screen readers--some information was not communicated expectedly and some was repeated.

You could consider whether it'd be better to just use a regular HTML select here as the current usecase for this select has a very limited set of options